<launch>

    <arg name="image_topic" default="/darknet_ros/detection_image"/>
    <arg name="output_image" default="/lane_images"/>
    <arg name="output_lane" default="/Lane_curvature"/>
    <arg name="weight_path" default="/home/choiin/lanenet_ws/src/lanenet_ros/model/tu_1976/tu.ckpt-1976"/>

    <arg name="use_gpu" default="1"/>
    <arg name="lane_image_topic" default="/lane_image"/>

    <node pkg="lanenet_ros" type="lanenet_ros_node.py" name="lanenet_node" output="screen">
        <param name="image_topic" value="$(arg image_topic)" />
        <param name="output_image" value="$(arg output_image)" />
        <param name="output_lane" value="$(arg output_lane)" />
        <param name="weight_path" value="$(arg weight_path)" />
        <param name="use_gpu" value="$(arg use_gpu)" />
        <param name="lane_image_topic" value="$(arg lane_image_topic)" />
    </node>
</launch>
